compiler/ballerina-parser/src/main/resources/syntax_diagnostic_message.properties

```diff
@@ -242,6 +242,8 @@ error.required.parameter.after.the.defaultable.parameter=required parameter afte
 error.named.arg.followed.by.positional.arg=named arg followed by positional arg
 error.arg.followed.by.rest.arg=arg followed by rest arg
 error.binding.pattern.not.allowed=binding pattern not allowed
+error.defaultable.parameter.cannot.be.included.record.parameter = defaultable parameter cannot be included \
```

Comment:
In [here](https://github.com/ballerina-platform/ballerina-lang/blob/e4d3b21058bb46e5d3a73e3ef3d909897ebf4e6f/compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java#L1761) I took the invalid node as `*`. So due to that, I set the error as `defaultable parameter cannot be included`. So if set the error as the mentioned way I want to change the invalid node as `=`.  Is there any special reason to make such a change?

Reply:
This was based on the assumption that when the user sees this error it's more likely that they'll continue to have it as an included param rather than a defaultable param. 

Given that it's just an assumption, let's go ahead with this.

But shall we say something like "a defaultable parameter cannot be an include parameter"?

Reply:
Changed.